File: TimelinesObjC/TimelinesObjC/LoginViewController.m
//
//  LoginViewController.m
//  TimelinesObjC
//
//  Created by Fabric on 5/26/15.
//  Copyright (c) 2015 Fabric. All rights reserved.
//

#import "LoginViewController.h"

@interface LoginViewController ()

@end

@implementation LoginViewController

- (IBAction)showTweetsTapped:(id)sender {
    [[Twitter sharedInstance] logInGuestWithCompletion:^(TWTRGuestSession *guestSession, NSError *error) {
        if (guestSession) {
            [self performSegueWithIdentifier:@"segueShowTweets" sender:self];
        } else {
            NSLog(@"error: %@", [error localizedDescription]);
        }
    }];
    
}

@end

File: TimelinesSwift/TimelinesSwift/LoginViewController.swift
//
//  ViewController.swift
//  TimelinesSwift
//
//  Created by Fabric on 5/26/15.
//  Copyright (c) 2015 Fabric. All rights reserved.
//

import UIKit
import TwitterKit

class LoginViewController: UIViewController {

    @IBAction func showTweetsTapped(sender: AnyObject) {
        Twitter.sharedInstance().logInGuestWithCompletion { guestSession, error in
            if (guestSession != nil) {
                self.performSegueWithIdentifier("segueShowTweets", sender: self)
            } else {
                println("error: \(error.localizedDescription)");
            }
        }
    }

}
